new-frontiers-14/frontier-station-14
# Better Harpy Mimicry

## About the PR


Harpies have a bunch of sound collections, but they currently aren't supported, since there's no accepted emote for them.  This adds the emotes.

Some emote types have whitelist or blacklists, but this PR adds an "mimicEmotes" field to SpeechComponents to allow whatever entities use it to bypass that check.  Now you can make your robotic noises and ~~annoy the SR~~ enjoy peak harpy roleplay.

## Why / Balance


Sound collections were there before, I believe these emotes used to work.

Issues:
* harpy emote wheel is pretty much unusable now.
* For reasons I am unaware of, certain emotes (sneezes, coughs, chirps) still don't play.

## How to test


1. Spawn in as a harpy.
2. Play the following emotes (e.g. *squishes):
    ```    - Squish
    - Chitter
    - Click
    - Ring
    - Pew
    - Bang
    - Rev
    - Caw
    - Sneeze # Frontier
    - Cough # Frontier
    - Hiss # Frontier
    - Meow # Frontier
    - Mew # Frontier
    - Growl # Frontier
    - Purrs # Frontier
    - Beep # Frontier
    - Squeak # Frontier
    - Chirp # Frontier
    - Snarl # Frontier
    - Bark # Frontier
    - Whine # Frontier
    - Howl # Frontier
    - Awoo # Frontier
    - Honk # Frontier
    ```
3. Note which emotes do not work.
4. Try to deathgasp (*deathgasp), it should not work.
5. Try to chime (*chimes) [not an accepted emote], it should not work.

## Media



## Breaking changes


**Changelog**



:cl:
- fix: More harpy emotes should be working (e.g. growls, honks)
